src/lib/features/feature-toggle/feature-strategies-read-model.ts

```diff
@@ -47,4 +46,60 @@ export class FeatureStrategiesReadModel implements IFeatureStrategiesReadModel {
               }
             : null;
     }
+
+    async getMaxConstraintValues(): Promise<{
+        feature: string;
+        environment: string;
+        count: number;
+    } | null> {
+        const rows = await this.db('feature_strategies')
+            .select(
+                'feature_name',
+                'environment',
+                'constraints',
+                this.db.raw(
+                    'MAX(jsonb_array_length(constraints)) as constraint_count',
+                ),
+                this.db.raw(
+                    "MAX(coalesce(jsonb_array_length(value->'values'), 0)) as max_values_count",
+                ),
+            )
+            .joinRaw('JOIN jsonb_array_elements(constraints) as value ON true')
```

Comment:
can we do it without join?
```
SELECT 
    feature_name, 
    environment, 
    constraints, 
    MAX(COALESCE(jsonb_array_length(value->'values'), 0)) AS max_values_count
FROM 
    feature_strategies,
    jsonb_array_elements(constraints) AS value
GROUP BY 
    feature_name, 
    environment, 
    constraints
ORDER BY 
    max_values_count DESC
LIMIT 2;
```

Reply:
Probably can! I'll look into it on monday morning 😄

Reply:
Yes we can! 3a39d55114f7bb1f4fbedd7f60d8ede147877503

Reply:
3a39d55114f7bb1f4fbedd7f60d8ede147877503